docs/using-pre-commit.md

```diff
@@ -27,35 +27,41 @@ Note that these hooks do not update the `.ipynb` notebook when you pull. Make su
 
 ## Using Jupytext with the pre-commit package manager
 
-Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file:
-```
+Using Jupytext with the [pre-commit package manager](https://pre-commit.com/) is another option. You could add the following to your `.pre-commit-config.yaml` file to convert all staged notebooks to python scripts in `py:percent` format (the default):
+
+```yaml
 repos:
--   repo: local
+-   repo: https://github.com/mwouts/jupytext
+    rev: master
     hooks:
     - id: jupytext
-      name: jupytext
-      entry: jupytext --to md
-      files: .ipynb
-      language: python
 ```
 
-Here is another `.pre-commit-config.yaml` example that uses the --pre-commit mode of Jupytext to convert all `.ipynb` notebooks to `py:light` representation and unstage the `.ipynb` files before committing.
-```
+You can also provide arguments to Jupytext in pre-commit, for example to produce several kinds of output files:
+
+```yaml
 repos:
-  -
-    repo: local
+-   repo: https://github.com/mwouts/jupytext
+    rev: master
     hooks:
-      -
-        id: jupytext
-        name: jupytext
-        entry: jupytext --from ipynb --to py:light --pre-commit
-        pass_filenames: false
-        language: python
-      -
-        id: unstage-ipynb
-        name: unstage-ipynb
-        entry: git reset HEAD **/*.ipynb
-        pass_filenames: false
-        language: system
+    - id: jupytext
+      args: [--to, py:light]
+    - id: jupytext
+      args: [--to, markdown]
+```
+
+If you are combining Jupytext with other pre-commit hooks, you must ensure that all hooks will pass on any files you generate. For example, if you have a hook for using `black` to format all your python code, then you should use Jupytext's `--pipe` option to also format newly generated Python scripts before writing them:
 
+```yaml
+repos:
+-   repo: https://github.com/mwouts/jupytext
+    rev: master
+    hooks:
+    - id: jupytext
+      args: [--to, py:percent, --pipe, black]
```

Comment:
Args should be none, (except maybe --sync). Also not all projects use black. These args should be flexible based on what the end user wants to do with it.

Reply:
Because I wasn't aware of it :) From discussion with @mwouts in the issue it sounds like best option is not to provide any default args and force the user to always choose